index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inspiration&family=Nanum+Gothic:wght@400;700;800&display=swap"
+    rel="stylesheet">
+
+  <link rel="stylesheet" href="css/style.css">
+  <title>Albert Einstein Tribute Page</title>
+</head>
```

Comment:
- [ ] Please include a `<meta name="viewport" content="width=device-width, initial-scale=1.0">` viewport element inside the `<head></head>` of your HTML page. So it will be like this:
    ```html
  <head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <!-- ... -->
  <title>Albert Einstein Tribute Page</title>
  </head>
  ```

   This is important for the browser responsiveness between different devices because it gives the browser instructions on how to control the page's dimensions and scaling. **You should include this tag on all your web pages.** 
   For more information about it, you can check this link: https://www.w3schools.com/css/css_rwd_viewport.asp